hs_core/views/__init__.py

```
@@ -292,6 +332,16 @@ def delete_resource(request, shortkey, *args, **kwargs):
     # deleting the metadata container object deletes the resource
     # so no need to delete the resource separately
     #res.delete()
+
+    # delete resource from HSAccess database
+    try:
+        ha_obj = HSAlib.HSAccess(str(request.user.username), 'unused', django_settings.HS_ACCESS_DB, django_settings.HS_ACCESS_USERNAME,
+                                 django_settings.HS_ACCESS_PASSWORD, django_settings.HS_ACCESS_HOST, django_settings.HS_ACCESS_PORT)
+        ha_obj.retract_resource(shortkey)
+    except:
```

Got it. Will change throughout my code to make sure no bare except. Thanks!